src/ngRoute/directive/ngView.js

```diff
@@ -19,8 +19,10 @@ ngRouteModule.directive('ngView', ngViewFillContentFactory);
  * Requires the {@link ngRoute `ngRoute`} module to be installed.
  *
  * @animations
- * enter - animation is used to bring new content into the browser.
- * leave - animation is used to animate existing content away.
+ * | Animation                        | Occurs                              |
+ * |----------------------------------|-------------------------------------|
+ * | {@link ng.$animate#enter enter}  | when the expression changes, on the new view |
+ * | {@link ng.$animate#leave leave}  | when the expression changes, on the old view |
```

Comment:
I would change the description to something like: _when the new/old view is inserted to/removed from the DOM_